Gemfile

```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 ruby '~> 3.2'
 
-gem 'rails', '~> 7.0.8'
+gem 'rails', '~> 7.1.3.3'
```

Comment:
im fine with 7.1

Reply:
Sorry, I meant whether going from `7.0` to `7.1` was intentional since it usually includes some other work described in the [upgrade guide](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-7-0-to-rails-7-1).

As an example, the IDP required a few other changes: https://github.com/18F/identity-idp/pull/9333

Reply:
updated!